public/index.html

```diff
@@ -24,7 +28,7 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
-    <title>React App</title>
+    <title>What's Near Me</title>
```

Comment:
Just wanted to say that this title works fine for now, but when y’all get to the last 2 weeks you are totally welcome to come up with your own, unique name!